rs/sns/governance/src/governance.rs

```diff
@@ -2603,12 +2609,72 @@ impl Governance {
 
     async fn perform_execute_extension_operation(
         &self,
-        _execute_extension_operation: ExecuteExtensionOperation,
+        execute_extension_operation: ExecuteExtensionOperation,
     ) -> Result<(), GovernanceError> {
-        Err(GovernanceError::new_with_message(
-            ErrorType::InvalidCommand,
-            "ExecuteExtensionOperation is not supported yet.",
-        ))
+        let ValidatedExecuteExtensionOperation {
+            extension_canister_id,
+            operation_name,
+            operation_arg,
+        } = execute_extension_operation.try_into().map_err(|err| {
+            GovernanceError::new_with_message(
+                ErrorType::InvalidProposal,
+                format!("Invalid ExecuteExtensionOperation proposal: {:?}", err),
+            )
+        })?;
+
+        validate_execute_extension_operation(
```

Comment:
In theory this is already validated ahead of time?  Are there things that can change between proposal creation and execution?